test/integration/test_kraken.py

```diff
@@ -173,3 +175,25 @@ def test_kraken_krona(tmpdir, kraken_db, krona_db, input_bam):
     parser = metagenomics.parser_krona(argparse.ArgumentParser())
     args = parser.parse_args([out_reads, krona_db, out_html])
     args.func_main(args)
+
+
+
+@pytest.mark.skipif(tools.is_osx(), reason="kraken osx binary does not yet exist on bioconda")
+def test_kraken_on_empty(kraken_db, input_bam):
```

Comment:
Why didn't the remaining tests move over as well? Those all failed at some point before we hardened the code for empty data handling.